lib/statsd/instrument/assertions.rb

```diff
@@ -11,6 +11,10 @@ def assert_statsd_increment(metric_name, options = {}, &block)
     assert_statsd_call(:c, metric_name, options, &block)
```

Comment:
Not sure, but if we want to go with the new API, then maybe this old method should now just call the new one (with `ordered_metric_options = [ options ]`?)